.github/workflows/update_website.yml

```diff
@@ -2,21 +2,26 @@ name: Update Group Website
 
 on:
   workflow_dispatch:
-  workflow_run:
-     workflows: ["Check Webhook Dispatch"]
-     types: 
-       - completed
+  repository_dispatch:
+    types: [repo-push]
   push:
-     branches:
-       - main
-     paths-ignore:
-       - 'group-data/**'
-: '
+    branches:
+      - main
+
 jobs:
-  Update_website:
+  update_website:
     name: Updating website
     runs-on: ubuntu-latest
     steps:
+      - name: Set up Python3
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.12"
+
+      - name: Update pip and install Jupyter and Pandas
+        run: |
+          pip install --upgrade pip jupyter pandas
```

Comment:
Thanks for all the changes. One last change- can you do git clone first, then setup python and install dependencies, then run the noteboooks